src/input_processing/dual_axis/mod.rs

```diff
@@ -205,133 +205,487 @@ pub trait WithDualAxisProcessingPipelineExt: Sized {
         self.with_processor(DualAxisBounds::symmetric_all(threshold))
     }
 
+    /// Appends a [`DualAxisBounds`] processor as the next processing step,
+    /// restricting values to a minimum value on both axes.
+    #[inline]
+    fn at_least(self, min: f32) -> Self {
+        self.with_processor(DualAxisBounds::at_least_all(min))
+    }
+
+    /// Appends a [`DualAxisBounds`] processor as the next processing step,
+    /// restricting X values to a minimum value.
+    #[inline]
+    fn at_least_only_x(self, min: f32) -> Self {
```

Comment:
This name might be a bit confusing, but I'm having trouble coming up with a better one